Gemfile.lock

```diff
@@ -1,4 +1,24 @@
 GIT
+  remote: git://github.com/Dantheman720/aweplug.git
+  revision: b73a61660f407eb8c7e76d69c395ab06aed8d0c9
```

Comment:
You are not testing with the right commit. You should have this one: https://github.com/Dantheman720/aweplug/commit/21591e79452685ee3728b2d23ec885e811e1f75f